### PR TITLE
Sec agent

### PR DESCRIPTION
**Pull request by matas-cast**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -71,3 +71,9 @@ variable "default_node_configuration" {
   type        = string
   description = "ID of the default node configuration"
 }
+
+variable "install_security_agent" {
+  type = bool
+  default = false
+  description = "Optional flag for installation of security agent"
```
Comment (aldor007) [inline variables.tf:78]: Can you add a link to the doc where the security agent is described?

File: main.tf [hunk 2]
```diff
@@ -230,3 +230,29 @@ resource "castai_autoscaler" "castai_autoscaler_policies" {
...
+
+  name             = "castai-sec-agent"
+  repository       = "https://castai.github.io/helm-charts"
+  chart            = "castai-sec-agent"
+  namespace        = "castai-agent"
+  create_namespace = true
+  cleanup_on_fail  = true
+
+  set {
+    name  = "castai.apiURL"
```
Comment (aldor007) [inline main.tf:245]: No need for an API token?

Comment (matas-cast) [reply main.tf:245]: we will reuse cluster controller secret

File: variables.tf (hunk 1 above)
Comment (aldor007) [reply variables.tf:78]: @matas-cast  can you regenerate doc?

**Review by aldor007 — APPROVED** [review]

**Pull request merged**